input/02-3-wijziging-gebiedsaanwijzing/main.json

```diff
@@ -116,6 +116,7 @@
     },
     "regeling_vrijetekst": "./regelingvrijetekst_template.xml",
     "regeling_mutatie": {
+        "type": "vervang",
```

Comment:
You can change the type to `renvooi` and then also fill in the key below to get the renvooi service to work